db/migrate/20240612102426_add_constraints_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddConstraintsToUsers < ActiveRecord::Migration[7.1]
+  def change
+    change_column_null :users, :name, false
+    change_column_null :users, :email, false
+    change_column_null :users, :password_digest, false
```

Comment:
Why do we need seperate migration file to change column?

Reply:
Bhai, i forgot to add necessary constraints on 1st migration, So I created 2nd migration to add the constraints.